isteven-multi-select.js

```diff
@@ -1208,22 +1216,21 @@ angular
           // watch1, for changes in input model property
           // updates multi-select when user select/deselect a single checkbox programatically
           // https://github.com/isteven/angular-multi-select/issues/8
-          $scope.$watch(
+          $scope.$watchCollection(
```

Comment:
This performs a shallow comparison instead of deep, huge performance difference.